tests/PHPStan/Rules/Properties/TypesAssignedToPropertiesRuleTest.php

```diff
@@ -43,10 +43,6 @@ public function testTypesAssignedToProperties(): void
 				'Static property PropertiesAssignedTypes\Foo::$staticStringProperty (string) does not accept int.',
 				37,
 			],
-			[
-				'Property PropertiesAssignedTypes\Ipsum::$parentStringProperty (string) does not accept int.',
```

Comment:
`parent::$parentStringProperty` is not valid since the property is an instance one.

`$this->parentStringProperty` is correctly reported